runtime/js-compute-runtime/builtins/backend.cpp

```diff
@@ -1120,6 +1121,31 @@ JSObject *Backend::create(JSContext *cx, JS::HandleObject request) {
 
   JS::SetReservedSlot(backend, Backend::Slots::DontPool, JS::BooleanValue(false));
 
+  if (Fastly::defaultDynamicBackendConfig.connect_timeout.has_value()) {
+    JS::RootedValue connect_timeout_val(
+        cx, JS::NumberValue(Fastly::defaultDynamicBackendConfig.connect_timeout.value()));
+    if (!Backend::set_timeout_slot(cx, backend, connect_timeout_val, Backend::Slots::ConnectTimeout,
+                                   "connectTimeout")) {
+      return nullptr;
+    }
+  }
```

Comment:
Why does this unconditionally set the values if the defaults are present? Will they be overridden by user code later?

Reply:
This function creates a dynamic backend for a request's host, and this function doesn't currently support setting timeouts at all, since timeouts are only currently supported for explicitly created backends that are passed in.

The default timeouts are therefore always applied when present, providing a way to set them for this use case.